src/use-document-query-command-state.hook.js

```diff
@@ -3,7 +3,7 @@ import {RichTextContext} from './rich-text-container.component.js'
 
 const defaultActiveInfo = {isActive: false, value: false}
 
-export function useDocumentQueryCommandState(commandName) {
+export function useDocumentQueryCommandState(commandName, tagName) {
```

Comment:
I think this would make more sense if named `activeValueMatch` or something? Since `document.queryCommandValue()` does not always return a tagName.